R/utilities-simulation.R

```diff
@@ -25,7 +25,7 @@
 #' @param stopIfParameterNotFound Logical. If `TRUE` (default), an error is
 #'   thrown if any of the `additionalParams` does not exist. If `FALSE`,
 #'   non-existent parameters are  ignored.
-#'  @param simulationRunOptions Optional instance of a `SimulationRunOptions`
+#' @param simulationRunOptions Optional instance of a `SimulationRunOptions`
```

Comment:
Can you please also update the `.Rd` file for this change?

Reply:
Done